api/src/opentrons/protocol_engine/commands/calibration/move_to_location.py

```diff
@@ -0,0 +1,124 @@
+"""Calibration Move To Location command payload, result, and implementation models."""
+from __future__ import annotations
+from pydantic import BaseModel, Field
+from typing import TYPE_CHECKING, Type, Optional
+from typing_extensions import Literal
+from enum import Enum
+
+from opentrons.protocol_engine.commands.pipetting_common import PipetteIdMixin
+from opentrons.protocol_engine.types import DeckPoint
+from opentrons.types import DeckSlotName
+from opentrons.protocol_engine.commands.command import (
+    AbstractCommandImpl,
+    BaseCommand,
+    BaseCommandCreate,
+)
+
+if TYPE_CHECKING:
+    from opentrons.protocol_engine.execution import MovementHandler
+    from opentrons.protocol_engine.state.state import StateView
+
+
+MoveToLocationCommandType = Literal["calibration/moveToLocation"]
+
+
+class CalibrationPositions(str, Enum):
+    """Deck slot to move to."""
+
+    PROBE_POSITION = "probePosition"
+    ATTACH_OR_DETACH = "attachOrDetach"
+
+    @property
+    def offset(self) -> DeckPoint:
+        """Return offset values for the given position."""
+        if self.value == "probePosition":
+            return DeckPoint(x=10, y=0, z=3)
+        else:
+            return DeckPoint(x=0, y=0, z=0)
+
+
+class MoveToLocationParams(PipetteIdMixin):
+    """Calibration set up position command parameters."""
+
+    deckSlot: CalibrationPositions = Field(
+        ...,
+        description="Slot location to move to before starting calibration.",
+    )
+
+
+class MoveToLocationResult(BaseModel):
+    """Result data from the execution of a CalibrationSetUpPosition command."""
+
+    position: DeckPoint = Field(
+        ..., description="Deck Point position after the move command has been executed"
```

Comment:
Would this be clearer (and correct)?

```suggestion
        ..., description="Position in deck coordinates after this movement has been executed"
```

In order to clarify that these are global coordinates, not local coordinates relative to the deck slot or anything.